### ISSUE TITLE
16

### ISSUE THREAD

**Issue by halj3317-bit** [halj3317-bit/2#5]
import pandas as pd
from sklearn.preprocessing import LabelEncoder

df = pd.read_csv("/mnt/data/day16_encoding.csv")

print("1- عرض البيانات")
print(df.head())

le = LabelEncoder()
df["city_label"] = le.fit_transform(df["city"])

print("2- بعد label encoding")
print(df[["city","city_label"]].head())

df2 = pd.get_dummies(df, columns=["city"])

print("3- بعد one hot encoding")
print(df2.head())

print("4- مقارنة عدد القيم")
print(df["city_label"].value_counts())
print(df2.filter(like="city_").sum())
